packages/gatsby-source-strapi/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Up date this file to match babel rc files in other packages. Basically just use the Gatsby package preset